examples/todo-list/README.md

```diff
@@ -27,23 +27,30 @@ Lastly, you'll need to install the LoopBack 4 CLI toolkit:
 npm i -g @loopback/cli
 ```
 
+If you're not following from the
+[todo tutorial](http://loopback.io/doc/en/lb4/todo-tutorial.html), you can use
+the LoopBack 4 CLI tool to catch up to where this tutorial will continue from:
```

Comment:
@shimks Hmm...I find a little hard to understand the guide here: 
Do you mean people should follow the `todo` tutorial then continue with this one?